docs/_components/modal.md

```diff
@@ -93,8 +96,9 @@ Toggle a modal with a link (or a link button) by using the `data-toggle="modal"`
     id="myModal"
     tabindex="-1"
     role="dialog"
+    aria-modal="true"
     aria-labelledby="myModalLabel"
```

Comment:
Should this be `myModal3-title`? Personally, I'd stick to camelCase for this and `myModal-description` - so `myModalTitle` and `myModalDescription`.

Reply:
I don't know really. Whatever it feels more natural to you. If it was my call I would keep it more human readable and not camelCase